.github/ISSUE_TEMPLATE/0-bug-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+---
+
+## System information
+- CPU information(`cat /proc/cpuinfo | grep "model name" | uniq`):
+- GPU information(`lspci -nn | grep -E 'VGA|isplay`):
+- Display server if rendering to display(X or wayland):
+
+## Issue behavior
+### Describe the current behavior
+
+### Describe the expected behavior
```

Comment:
And we can put the BKM of dumping libva trace and driver log to this document. Request the bug submitter to provide those logs, then it will help us to debug and triage.

Reply:
Thanks for suggestion. Added libva trace log.